### PR TITLE
A detailed docs for Builtins.remove.

### PR DESCRIPTION
**Pull request by mvidner**

### CODE REVIEW

File: src/ruby/yast/builtins.rb [hunk 1]
```diff
@@ -146,9 +146,18 @@ def self.maplist(object, &block)
...
-    # - Remove item from term
+    # Remove *element* from a **copy** of *object*.
+    #
+    # @param object [::Hash,::Array,Yast::Term,nil] a container
+    # @param element [Object,Integer,nil]
+    #   Any key for Hash;
+    #   Integer for Array and Term;
+    #   terms are indexed from 1 (**one**)
+    # @return [Object,nil]
+    #   Returns *object* unchanged if either *object* or *element* is `nil`;
```
Comment (jreidinger) [inline src/ruby/yast/builtins.rb]: this is not true as it returns copy if element is nil. So not unchanged. For nil object it it not interesting as nil is immutable.

File: src/ruby/yast/builtins.rb [hunk 2]
```diff
@@ -146,9 +146,18 @@ def self.maplist(object, &block)
...
 
-    # - Removes element from a list
-    # - Remove key/value pair from a map
-    # - Remove item from term
+    # Remove *element* from a **copy** of *object*.
+    #
+    # @param object [::Hash,::Array,Yast::Term,nil] a container
+    # @param element [Object,Integer,nil]
+    #   Any key for Hash;
+    #   Integer for Array and Term;
```
Comment (jreidinger) [inline src/ruby/yast/builtins.rb]: maybe document that for negative number it returns copy of container without modification, so nothing like -1 is last or like that

File: src/ruby/yast/builtins.rb (hunk 1 above)
Comment (mvidner) [inline src/ruby/yast/builtins.rb]: Ah! Thanks for the catch.

**Comment by mvidner** [comment]
Updated.

**Comment by jreidinger** [comment]
LGTM

**Pull request merged**